apps/api-dispute/src/extract_schema.py

```diff
@@ -1,32 +1,34 @@
-import argparse
```

Comment:
Did you make changes here? I don't have time to check manually at the moment.

Reply:
The changes are importing `os` for creating the destination directory if it does not exist. But every line are changed 😕